packages/typescript/test/class.test.tsx

```diff
@@ -358,3 +358,22 @@ it("renders a method with parameter docs", () => {
     }
   `);
 });
+
+it("creates a nullish symbol for an optional member", () => {
```

Comment:
This test doesn't actually validate that a nullish symbol is created.

Reply:
Oh yeah moved these to MemberExpression